incubator/virtualcluster/pkg/syncer/resources/priorityclass/controller.go

```diff
@@ -18,6 +18,7 @@ package priorityclass
 
 import (
 	"fmt"
+	"sigs.k8s.io/multi-tenancy/incubator/virtualcluster/pkg/syncer/constants"
```

Comment:
Can you move this line back to previous location?

Reply:
sure